Home/dashboard.php

```diff
@@ -0,0 +1,25 @@
+<h1 class='welcomeMessage'>Welcome back User</h1>
+<div class="container">
+    <div id="homePageTickets">
+        <div class="homeDashboard">
+            <span class="homeProject-list">
+                <p><b>Your projects</b></p>
+                <hr class="small-hr">
+                <ul class="list-inline">
+                    <li class="list-inline-item">Lorem ipsum</li>
+                    <li class="list-inline-item">Phasellus iaculis</li>
+                    <li class="list-inline-item">Nulla volutpat</li>
+                </ul>
+            </span>
+            <span class="ticket-deadline">
+                <p><b>Ticket Deadlines</b></p>
+                <hr class="small-hr">
+                <ul class="list-inline">
+                    <li class="list-inline-item">Lorem ipsum</li>
+                    <li class="list-inline-item">Phasellus iaculis</li>
+                    <li class="list-inline-item">Nulla volutpat</li>
+                </ul>
+            </span>
+        </div>
+    </div>
+</div>
```

Comment:
Looks good, but the 'Your projects' section is coming off the page. I have attached a screenshot below.
<img width="1440" alt="Screenshot 2020-12-23 at 19 27 27" src="https://user-images.githubusercontent.com/43154254/103031018-ea92f100-4554-11eb-9b76-dff321c7998c.png">